### PR TITLE
docs: created readme.md file

### LINKED ISSUE #5: Readme
Nosso projeto iniciou agora e precisamos de um README explicando um pouco mais sobre o projeto e coisas afim use [o README do octopost](https://github.com/hatcoders/octopost/blob/master/README.md) como referência.

### PR DESCRIPTION
**Pull request by RafaelLimaC**
Closes #5 

created readme.md file

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>

image

  </summary>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+   - Faça um fork do repositório e trabalhe em novas funcionalidades.
+   - Resolva problemas existentes ou proponha melhorias.
+2. **Documentação:**
+   - Aprimore a documentação existente ou crie tutoriais para ajudar outros desenvolvedores.
+3. **Testes:**
+   - Ajude a garantir a estabilidade do aplicativo testando as novas funcionalidades e relatar problemas.
+
+## **Cargos**
+
+Em nosso [servidor do discord](https://discord.com/invite/K2ghxhP6ek), temos várias opções de cargos, que você poderá adquirir participando do servidor e subindo de level, contribuindo com o projeto.
```
Comment (GabrielTrindadeC) [inline README.md:51]: http://discord.gg/3gsMAEumEd

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- **`Bolha `** → Dev influente e conhecido da bolha dev e participa do servidor;
+- **`Dono do parquinho `** → Auto-explicativo. Idealizador do projeto e com poderes especiais.
+
+## **Código de Conduta**
+
+O PetDex segue um Código de Conduta para garantir um ambiente respeitoso e inclusivo para todos os contribuidores. Leia e entenda o nosso **[Código de Conduta](https://github.com/Alecell/octopost/blob/master/CODE_OF_CONDUCT.md)**.
+
+## **Contato**
+
+Se tiver alguma dúvida ou precisar de ajuda, fique à vontade para entrar em contato com nossa equipe em nosso [Discord](https://discord.com/invite/K2ghxhP6ek).
```
Comment (GabrielTrindadeC) [inline README.md:69]: http://discord.gg/3gsMAEumEd

File: README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+[![Docker](https://img.shields.io/badge/Docker-2496ED?logo=docker&logoColor=white)](https://www.docker.com/)
+[![JavaScript](https://img.shields.io/badge/JavaScript-323330?logo=javascript&logoColor=F7DF1E)](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript)
+[![HTML](https://img.shields.io/badge/HTML5-E34F26?logo=html5&logoColor=white)](https://developer.mozilla.org/pt-BR/docs/Web/HTML)
+[![Sass](https://img.shields.io/badge/Sass-FF69B4?logo=sass&logoColor=white)](https://sass-lang.com/)
+[![Jest](https://img.shields.io/badge/Jest-red?logo=jest&logoColor=white)](https://jestjs.io/)
+[![GitHub Contributors](https://img.shields.io/github/contributors/devhatt/pet-dex-frontend)](https://github.com/devhatt/pet-dex-frontend/contributors)
+[![GitHub Stars](https://img.shields.io/github/stars/devhatt/pet-dex-frontend.svg?style=social&label=Star&maxAge=2592000)](https://github.com/devhatt/pet-dex-frontend/stargazers/)
+[![Discord Server](https://img.shields.io/discord/308323056592486420?color=7289DA&label=Discord&logo=discord&logoColor=white)](https://discord.gg/9f5BZ7yD)
```
Comment (GabrielTrindadeC) [inline README.md:12]: http://discord.gg/3gsMAEumEd

File: README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+[![Twitter Follow](https://img.shields.io/twitter/follow/alecell_?style=social)](https://twitter.com/alecell_)
```
Comment (GabrielTrindadeC) [inline README.md:13]: https://twitter.com/DevHatt

**Review by GabrielTrindadeC — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+## **Introdução**
+
+O PetDex é um projeto mobile/desktop com o objetivo de facilitar a adoção de pets. No petdéx usuários e ONGs podem cadastrar os animais para adoção e o aplicativo vai fazer um sistema de match com o usuário que mais combina com aquele pet.
+
+## **Primeiros Passos**
+
+Para começar no PetDex, siga estes passos:
+
+1. Clonar o repositório: **`git clone https://github.com/devhatt/pet-dex-frontend.git`**
```
Comment (GabrielTrindadeC) [inline README.md:27]: adicionar o passo pra pessoa entrar na pasta clonada antes de instalar as dependências

**Review by GabrielTrindadeC — CHANGES_REQUESTED** [review]

**Review by GabrielTrindadeC — APPROVED** [review]

**Comment by GabrielTrindadeC** [comment]
LGTM!

**Review by aalmeida00 — APPROVED** [review]

**Pull request merged**